docs/rule/link-href-attributes.md

```diff
@@ -1,6 +1,10 @@
 # link-href-attributes
 
-It's common to treat anchor tags as buttons. However, this is typically considered bad practice, as an anchor tag without an `href` is unfocusable which breaks accessibility.
+It's common to treat anchor tags as buttons. However, this is typically considered bad practice, as an anchor tag without an `href` is unfocusable which breaks accessibility. The most discernable difference between a link (`<a>`) and a `<button>` is that a link navigates the user to a new URL (thus taking the user away from the current context). A `button` toggles something in the interface or triggers new content in that same context (i.e., a popup menu using `aria-haspopup`).
+
+One of the differences between `<a>` elements and `<button>` elements is how they work- the `<button>` can be triggered by either the `SPACEBAR` or the `ENTER` key, while the `<a>` is only triggered by the `ENTER` key. This is important to know, so you can provide appropriately for users who rely UI elements to work in a consistent way. 
+
+Wherever possible, a link should look like a link, and a button should look like a button. However, the underlying HTML markup and base functionality are more important here (if one has to choose). 
 
 ## Examples
```

Comment:
Could you add the following example to help demonstrate what to do when you want to send an action?

```hbs
<button type="button" {{on 'click' this.submit}}>Submit</button>
```